src/menu/views/cpakfs_manager.c

```diff
@@ -0,0 +1,970 @@
+#include <stdbool.h>
+#include <stdio.h>
+#include <libdragon.h>
+#include "views.h"
+#include "../sound.h"
+#include "../fonts.h"
+#include <fatfs/ff.h>
+#include <errno.h>
+#include <dir.h>
+#include "utils/cpakfs_utils.h"
+
+#define MAX_STRING_LENGTH 62
+
+#define CPAK_EXTENSION ".mpk"   
+#define CPAK_NOTE_EXTENSION ".mpkn"
```

Comment:
Use `pak.notes`?

Reply:
I am not going to enforce in this PR... Though, just noting it just might be changed after.